package.json

```diff
@@ -12,45 +12,65 @@
     "ionic:serve": "ionic-app-scripts serve"
   },
   "dependencies": {
-    "@angular/animations": "^4.2.5",
-    "@angular/common": "4.2.4",
-    "@angular/compiler": "4.2.4",
-    "@angular/compiler-cli": "4.2.4",
-    "@angular/core": "4.2.4",
-    "@angular/forms": "4.2.4",
-    "@angular/http": "4.2.4",
-    "@angular/platform-browser": "4.2.4",
-    "@angular/platform-browser-dynamic": "4.2.4",
-    "@ionic-native/contacts": "^3.13.1",
-    "@ionic-native/core": "3.12.1",
-    "@ionic-native/splash-screen": "3.12.1",
-    "@ionic-native/status-bar": "3.12.1",
+    "@angular/animations": "^4.1.3",
```

Comment:
Right, `4.1.3` is the angular version generated by the current latest ionic cli. I had many difficulties getting our project to work with the emulator, it was throwing non-sense errors. finally I got it to work, downgrading angular version was the fix. these issues could be caused by npm 5 or ionic. I can't remember them in details! but there are couple of opened issues related to it. like [this one](https://github.com/ionic-team/ionic/issues/12357)